test-example-diagrams/src/main/resources/multiinstances-callactivity/MultiInstancesCallActivity.md

```diff
@@ -0,0 +1,24 @@
+# Multi Instance Service Task
```

Comment:
The header should be **Multi Instance Call Activity**

Reply:
Done